codemeta.json

```diff
@@ -0,0 +1,115 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "name": "FairRootGroup/FairMQ",
+    "description": "C++ Message Queuing Library and Framework",
+    "license": "https://spdx.org/licenses/LGPL-3.0-or-later",
+    "version": "v1.3.9",
+    "datePublished": "2019-08-07",
+    "contributor": [
+        {
+            "@type": "Person",
+            "familyName": "Alexey Rybalchenko"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Dennis Klein",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "GSI Helmholtz Centre for Heavy Ion Research GmbH"
+            }
+        },
+        {
+            "@type": "Person",
+            "familyName": "Nicolas Winckler",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Bull-Atos"
```

Comment:
How is this meant to be updated/maintained when someone's affiliation changes?

Reply:
* I'll take that question to the next ESCAPE meeting. Seems interesting genereally.
* First quick answer: The same as on zenodo: Find someone to update it. Not very scalable.
* Also maybe the meaning is "The affiliation that person had, when working on the project"? I'll try to figure that out.
* One possible way around this: Don't add an affiliation. Just add an ORCID. You can update your own ORCID.